quickstarts/dotnet/dnn-evoq/config.yml

```diff
@@ -3,19 +3,31 @@ name: dnn-evoq
 description: |
   ## What is DNN Evoq?
 
-  Evoq is the commercial / enterprise edition of DNN, the largest and most popular 
-  open source CMS on the Microsoft ASP.NET stack.
+  DNN Evoq is a commercial edition of the DNN platform, the largest and most popular open source Content Management System (CMS) on the Microsoft ASP.NET stack. [DNN Evoq](https://developer.newrelic.com/instant-observability/dnn-evoq/82672f5f-48ec-477c-ae49-7eb24a91595f/) includes customer support and provides premium features not available in the DNN platform. These premium features include personalization, in-page analytics, workflow and built-in connectors to Marketo, SharePoint, Amazon S3, and Dropbox.
```

Comment:
I'm not sure why this is linking to itself, isn't this the quickstart link for the site for DNN Evoq?

Reply:
Yes it is, I asked the writers to use that to review not to link. My apologies. Please remove the link for now and we'll find a better option in the future to link it to. Right now there's not an obvious choice.